vision/segmentation/plane_segmentation_test.go

```diff
@@ -51,6 +53,12 @@ func TestPlaneConfig(t *testing.T) {
 }
 
 func TestSegmentPlaneWRTGround(t *testing.T) {
+	// Setting a global, so cannot use t.Parallel()
+	sortPositions = true
+	defer func() {
+		sortPositions = false
+	}()
```

Comment:
I did this in all the tests of this file, since it was originally a global, but I'm not sure it's all needed.